examples/spark/examples_utils.py

```diff
@@ -92,11 +97,15 @@ def get_dataset_attrs(name: str):
     )
 
 
-def prepare_test_and_train(spark: SparkSession, path: str, seed: int) -> Tuple[SparkDataFrame, SparkDataFrame]:
+def prepare_test_and_train(spark: SparkSession, path: str, seed: int, is_csv: bool = True) -> Tuple[SparkDataFrame, SparkDataFrame]:
```

Comment:
more extending way to handle other formats is to parameter `format='parquet|cvs', **format_kwargs`